db/migrate/20160909090711_remove_char_limit_for_remote_job_invocation.rb

```diff
@@ -0,0 +1,10 @@
+class RemoveCharLimitForRemoteJobInvocation < ActiveRecord::Migration
+  def up
+    change_column :template_invocation_input_values, :value, :string, :null => false, :limit => nil
+    change_column :job_invocations, :description, :string, :limit => nil
```

Comment:
These should at least be type `:text` instead.

As far as extending the length of job description, why? I think it's not particularly related, descriptions should be fairly short and truncated.